express/code/blocks/pricing-cards/pricing-cards.css

```diff
@@ -91,7 +91,7 @@ Border style variants
 }
 
 .pricing-cards .gradient-promo {
-  background: linear-gradient(90deg, #ff477b 0%, #5c5ce0 52%, #318fff 100%) !important;
+  background: var(--S2-Buttons-Premium) !important;
```

Comment:
pricing-cards-simplified? I don't see pricing-cards anywhere in the test url DOM